src/pool/srv_pool.c

```diff
@@ -2170,7 +2170,7 @@ ds_pool_create_handler(crt_rpc_t *rpc)
 	ABT_rwlock_unlock(svc->ps_lock);
 	rdb_tx_end(&tx);
 	if (rc != 0)
-		D_GOTO(out_svc, rc);
+		D_GOTO(out_mutex, rc);
```

Comment:
Is the D_GOTO(out_svc, rc) on line 2189 in the conditional below also needing a change to D_GOTO(out_mutex, rc) ?

Reply:
Good catch. I don't know how I missed that one.